pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -174,6 +179,19 @@ async def _get(self, path: str, timeout: int | None = None) -> Any:
         ):
             return await _deserialise(response)
 
+    async def _get_text(self, path: str, timeout: int | None = None) -> Any:
+        # Optionally override default http timeout
+        http_timeout = timeout or self.http_timeout
+        async with (
+            aiohttp.ClientSession() as session,
+            session.get(
+                f"{self._url}{path}",
+                auth=self._auth,
+                timeout=http_timeout,
+            ) as response,
+        ):
+            return await response.text()
```

Comment:
the `f"/queries/{query_id}/modality"` endpoint returns plain text, and so an error is raised if you try to deseralise the response